README.md

```diff
@@ -75,17 +75,24 @@ How many riders represent team Ducati?
 
 ## Building
 
-### Linux Ubuntu 16.04
+### Compiling
 
 Requirements:
 
 * The RedisGraph repository: `git clone https://github.com/RedisLabsModules/RedisGraph.git`
-* The build-essential and cmake packages: `apt-get install build-essential cmake`
+* Python 2.7
```

Comment:
I think we should be ok with python 3 aswell but I might be wrong, need to verify.

Reply:
Actually, all new Python code is version agnostic. Pipenv works the same for P2 and P3, as well as the setup-*.py files.